pkg/providers/config.go

```diff
@@ -27,10 +27,9 @@ var DefaultConfigNamespace, _ = k8sutil.GetWatchNamespace()
 
 //DeploymentStrategyMapping Deployment Strategy Map
 type DeploymentStrategyMapping struct {
```

Comment:
in this directory can we update ` buildDefaultConfigMap()`

Reply:
Yep, missed that